mmv1/products/parallelstore/Instance.yaml

```diff
@@ -0,0 +1,147 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/instances
+create_url: projects/{{project}}/locations/{{location}}/instances?instanceId={{instance_id}}
+self_link: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+id_format: projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/instances/{{instance_id}}
+name: Instance
+description: A Parallelstore Instance.
+update_verb: :PATCH
+update_mask: true
+min_version: beta
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'parallelstore_instance_basic'
+    primary_resource_id: 'instance'
+    min_version: 'beta'
+    vars:
+      name: 'instance'
+      network_name: 'network'
+      address_name: 'address'
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: "The resource name of the instance, in the format\n`projects/{project}/locations/{location}/instances/{instance_id}` "
+    output: true
+  - !ruby/object:Api::Type::String
+    name: description
+    description: 'The description of the instance. 2048 characters or less. '
+  - !ruby/object:Api::Type::String
+    name: state
+    description: "The instance state. \n Possible values:\n STATE_UNSPECIFIED\nCREATING\nACTIVE\nDELETING\nFAILED"
+    output: true
+  - !ruby/object:Api::Type::String
+    name: createTime
+    description: 'The time when the instance was created. '
+    output: true
+  - !ruby/object:Api::Type::String
+    name: updateTime
+    description: 'The time when the instance was updated. '
+    output: true
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: labels
+    api_name:
```

Comment:
`api_name` is for situations where we want the field's name in the provider resource to not match the name of the field in the API. Do you need to rename a field from the API to become `labels`? Or this might need deleting.